src/sentry/incidents/tasks.py

```diff
@@ -200,54 +195,3 @@ def auto_resolve_snapshot_incidents(alert_rule_id, **kwargs):
         auto_resolve_snapshot_incidents.apply_async(
             kwargs={"alert_rule_id": alert_rule_id}, countdown=1
         )
-
-
-@instrumented_task(
-    name="sentry.incidents.tasks.process_pending_incident_snapshots", queue="incident_snapshots"
-)
-def process_pending_incident_snapshots(next_id=None):
```

Comment:
Could we have messages in the queues for this task during a deploy? Should this method stay but as a no-op, or do we ignore any sentry errors that may come from this task vanishing?

Reply:
Hmm, I think I'm ok with a couple of errors firing for this task, shouldn't cause any actual problems.